modules/ROOT/pages/changelog.adoc

```diff
@@ -5,6 +5,18 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes/index.adoc[{productname} Release Notes].
 
+== 5.10.4 - 2022-04-27
+
+=== Fixed
+* Inline toolbars flickered when switching between editors.
+* Multiple inline toolbars were shown if focused too quickly.
+
+== 5.10.3 - 2022-02-09
```

Comment:
Oh I didn't realise this was missing as well. Thanks for pulling this content across as well! 🏆